src/HostedExplorer.tsx

```diff
@@ -25,8 +25,7 @@ initializeIcons();
 
 const App: React.FunctionComponent = () => {
   // For handling encrypted portal tokens sent via query paramter
-  const params = new URLSearchParams(window.location.search);
-  const [encryptedToken, setEncryptedToken] = React.useState<string>(params && params.get("key"));
+  const [encryptedToken, setEncryptedToken] = React.useState<EncryptedAccessToken>();
```

Comment:
Since we are no longer assigning an initial value for `encryptedToken`, in line 36, we should also remove the initial value for `authType` as the expression would always evaluate to `undefined`.